coil-base/src/main/java/coil/RealImageLoader.kt

```diff
@@ -474,7 +479,7 @@ internal class RealImageLoader(
         private var size: Size? = null
 
         @MainThread
-        suspend inline fun size(cached: BitmapDrawable? = null): Size = scope.run {
```

Comment:
Pretty sure this `inline` isn't actually being inlined because `var size` is private.